engines_json/fr.sudoc.json

```diff
@@ -0,0 +1,60 @@
+{
+		"_name": "SUDOC",
+		"_alias": "SUDOC",
+		"_description": "Rechercher dans le SUDOC",
+		"_icon": "http://www.sudoc.abes.fr/~c_psi/psi_images/img_psi/3.0/favicons/default.ico",
+		"_hidden": false,
+		"_urlTemplate": "http://www.sudoc.abes.fr/cbs//DB=2.1/SET=2/TTL=1/CMD?ACT=SRCHA&IKT=1016&SRT=RLV&TRM={z:title}+{rft:aufirst?}+{rft:aulast?}",
+		"_urlParams": [],
+		"_urlNamespaces": {
+			"rft": "info:ofi/fmt:kev:mtx:book",
+			"z": "http://www.zotero.org/namespaces/openSearch#",
+			"": "http://a9.com/-/spec/opensearch/1.1/"
+		},
+		"_iconSourceURI": "http://www.sudoc.abes.fr/~c_psi/psi_images/img_psi/3.0/favicons/default.ico"
+	},
+{
```

Comment:
You need, unfortunately, given the current build script, to create a separate `json` file for each of these templates.